list/list.mbt

```diff
@@ -1068,3 +1081,35 @@ test "is_suffix" {
   @assertion.assert_true(List::[1, 2, 3, 4, 5].is_suffix(List::[3, 4, 5]))?
   @assertion.assert_false(List::[1, 2, 3, 4, 5].is_suffix(List::[3, 4, 6]))?
 }
+
+/// Separated helper function for intersperse
+fn prepend_to_all[T](self : List[T], sep : T) -> List[T] {
```

Comment:
unused function?

Reply:
> unused function?

Yeah.. this should be removed along with the duplicated definition of `intersperse`.